src/wallet/fees.cpp

```diff
@@ -13,10 +13,14 @@
 CAmount GetRequiredFee(const CWallet& wallet, unsigned int nTxBytes)
 {
     CAmount nBaseFee = MIN_TX_FEE;
-    CAmount nMinFee = (1 + nTxBytes / 1000) * nBaseFee;
+    CAmount nMinFee = nTxBytes * nBaseFee / 1000;
+
+    if( nMinFee < MIN_TX_FEE )
```

Comment:
This doesn't look right to me. E.g. for a transaction with size=1000, this will return a min. fee of 0.2 VRM, but really the min. fee should be 0.4 VRM.

Reply:
Yes you are right @hd4r however still there is an issue, if the 1 was still there a 500 byte tx would be a fee of 0.3 which should only be 0.3

Reply:
I'd like to just use the former code but I know the new fee implementation in btc is quite fragmented now and has many conditions we don't need.

Reply:
I don't think it would be 0.3: because the variables are integers, an integer division should be used -> 500/1000 = 0

Reply:
And now ? :)

Reply:
@hd4r ah yes that's the logic I was missing didn't think about it being ints